static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
```

Comment:
@yyscamper I can change the list of accepted rules no problem.  

I was pointed to the following link to show the rules we use from anchor
https://github.com/sailsjs/anchor/blob/cbbc3a51f0b400d617a2def2002ce1c2bf210a4b/lib/match/rules.js

maybe @benbp can comment, but is there a chance we support all the anchor rules and our documentation just isn't updated?